packages/core/src/dialects/db2/query-generator.js

```diff
@@ -105,7 +105,7 @@ export class Db2QueryGenerator extends Db2QueryGeneratorTypeScript {
             const commentText = commentMatch[2].replace(/COMMENT/, '').trim();
             commentStr += _.template(commentTemplate, this._templateSettings)({
               table: this.quoteTable(tableName),
-              comment: this.escape(commentText, { replacements: options.replacements }),
+              comment: this.escape(commentText, { replacements: options?.replacements }),
```

Comment:
TBH supporting replacements here doesn't really make any sense, we could remove the option

Reply:
Agreed, have removed it